ai-ml/ray/terraform/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "region" {
+  description = "AWS Region"
+  type        = string
+  default     = "us-east-1"
```

Comment:
Are we not using `us-west-2` as default?

Reply:
fixed